ic-task-scheduler/tests/dummy_scheduler_canister/src/canister.rs

```diff
@@ -132,7 +135,7 @@ impl DummyCanister {
 
     fn do_run_scheduler() {
         let scheduler = SCHEDULER.with_borrow(|scheduler| scheduler.clone());
-        scheduler.run().unwrap();
+        scheduler.run(()).unwrap();
```

Comment:
Self::Ctx::default()

Reply:
Clippy warns about calling `default()` for unit type. Let's keep it as is.